grammar/rainerscript.c

```diff
@@ -2368,6 +2368,79 @@ doFuncCall(struct cnffunc *__restrict__ const func, struct svar *__restrict__ co
 		varFreeMembers(&r[0]);
 		break;
 	}
+	case CNFFUNC_IS_TIME: {
+		char *fmt = NULL;
+
+		cnfexprEval(func->expr[0], &r[0], usrptr, pWti);
+		str = (char*) var2CString(&r[0], &bMustFree);
+
+		// Check if the optional 2nd parameter was provided
```

Comment:
It would be good to do the "real" work not inside the switch, but rather in a dedicated function. We try to keep the code managable size. I am sorry I seem to have overlooked this in previous PRs.

Reply:
No problem. I'll do that with this one, then do a separate PR to get the other ones set up like that too.

Reply:
great, thx a lot! I admit I may have missed that myself as well... ;-)

Reply:
Done.